docs/src/faq.md

```diff
@@ -114,3 +114,13 @@ In case these operators are too complicated to code them yourself, here are a fe
 - [ProximalOperators.jl](https://github.com/JuliaFirstOrder/ProximalOperators.jl)
 
 An alternative is differentiating through the KKT conditions, which is exactly what [DiffOpt.jl](https://github.com/jump-dev/DiffOpt.jl) does for JuMP models.
+
+### Memoization
+
+In some cases, performance might be increased by using memoization to prevent redundant calls to `forward`.
+For instance, this is relevant when calculating large Jacobians with forward differentiation, where the computation happens in [chunks](https://juliadiff.org/ForwardDiff.jl/stable/user/advanced/#Configuring-Chunk-Size).
+Packages such as [Memoize.jl](https://github.com/JuliaCollections/Memoize.jl) and [Memoization.jl](https://github.com/marius311/Memoization.jl) are useful for defining a memoized version of `forward`:
+
+```julia
```

Comment:
which package one are you using here?

Reply:
I was thinking we might wanna specify the import in the example

Reply:
Sure, I have updated the PR